build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -40,6 +42,9 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
             Channel = c.BuildContext.Get<string>("Channel");
             CommitHash = c.BuildContext.Get<string>("CommitHash");
 
+            // Do not publish symbol packages on a release branch.
+            IncludeSymbolPackages = !c.BuildContext.Get<string>("BranchName").StartsWith("release/");
```

Comment:
It will prevent us from publishing prerelease symbol packages on a release branch, yes. I figure this might actually be a small benefit, because we'll be able to find anyone relying on MyGet for symbol packages just a bit earlier and have a chance to communicate the RTM way to get them. It also means we have more consistent published set of artifacts when a build starts making stable versions.

I could change the logic to default to branch name but also allow an override if you think publishing prerelease release-branch symbol packages will be useful. (Or it could be based on whether the branch is stable--but I haven't seen a good central source to determine that during the build yet.)